README.md

```diff
@@ -1,2 +1,81 @@
 This is Confluence of Conference.
+
+# Setup Guide
+
+### Creating a Virtual Environment
+Virtual Environment allows us to have requirements installed specifically to our project instead of complete system. To create a virtual environment goto any directory where you will store your code and run the following command on terminal (linux) or Windows PowerShell (Windows)(For windows, run Windows Powershell as Administrator before running below commands)
+```
+python -m venv venvconf
+```
+#### Activating the Virtual Environment
+Run the following command to activate your virtual environment:
+##### Windows
+```
+./venvconf/Scripts/activate
+```
+##### Linux
+```
+source venvconf/bin/activate
+```
+**Note:** To deactivate the Virtual Environment, type deactivate in the shell to deactivate.
+#### Clone your forked repo
+```
+git clone https://github.com/your_username/Confluence
+```
+##### Change to develop branch
+Code development should happed in 'develop' branch and not in 'master' branch
+```
+git checkout develop
+```
+
+#### Installing Project Requirements (First Time Only or on Requirement changes)
+Change directory to cloned directory 
+```
+cd confluence
+```
+Use pip to install  missing requirements: (if you don't have pip installed on your system, google how to install) (Windows Users should run this command Adminsitrator Shell or will get error)
+```
+pip install -r  .\requirements.txt
+```
+#### Creating Environment Variables File
+Change directory to src/confluence to create a ".env" file that will contain your enviroment variables. **Note:** This file will not ignored by git and will be available automatically on production environment.
+```
+cd src\confluence
+```
+Create file ".env" with following data:
+```
+EXPLARA_API_KEY=YOUR_EXPLARA_KEY_HERE
+FACEBOOK_PAGE_ACCESS_TOKEN=YOUR_PAGE_ACCESS_TOKEN_HERE
+TWITTER_CONSUMER_KEY=YOUR_TWITTER_CONSUMER_KEy_HERE
+TWITTER_CONSUMER_SECRET=YOUR_CONSUMER_SECRET_HERE
+TWITTER_ACCESS_KEY=YOUR_ACCESS_KEY_HERE
+TWITTER_ACCESS_SECRET=YOUR_ACCESS_SECRET_HERE
+```
+**Note:** Create your own apps to retrieve these keys for testing.
+
+#### Create Database Migrations
+
+Navigate to root of /src and run the following commands to make database tables.
+```
+python manage.py makemigrations
+python manage.py migrate
+```
+
+#### Run Server 
+Once all above setup is done, run the following code to start your development server,
+```
+python manage.py runserver
+```
+You will see something like
+```
+Performing system checks...
+
+System check identified no issues (0 silenced).
+June 16, 2017 - 23:46:33
+Django version 1.9, using settings 'confluence.settings'
+Starting development server at http://127.0.0.1:8000/
+Quit the server with CTRL-BREAK.
+```
+On successful setup you will see the second last line i.e. ' Starting Development Server at http://127.0.0.1:8000/ '. Visit this url on your web browser to see if server is started successfully. On failed startup, you will see a 'Refused to connect' error.
```

Comment:
It happens when Server is not started i.e. the runserver command get stucked at some point while starting server due to some error and the developer may think that server is started.